components/Blogpost.jsx

```diff
@@ -13,20 +16,29 @@ export default function Blogpost({ post, slice = false }) {
       direction="column"
     >
       <Grid item textAlign="right">
-        <Typography variant="h3" component="h1">
-          {title}
+        <Typography variant="h3" component={slice ? 'h2' : 'h1'}>
+          <Link href={`/blog/${id}`}>{title}</Link>
         </Typography>
         <Typography variant="caption">
           {format(new Date(created), 'MM/dd/yyyy hh:mm')}
         </Typography>
       </Grid>
       <Grid item>
-        <Typography paragraph={true}>
-          {slice
-            ? `${content.slice(0, 200)}${content.length > 200 ? '…' : ''}`
-            : content}
-        </Typography>
+        <StyledContent
+          slice={slice}
+          dangerouslySetInnerHTML={{ __html: content }}
+        />
       </Grid>
     </Grid>
   );
 }
+
+const StyledContent = styled.div`
+  ${({ slice }) =>
+    slice &&
+    `
```

Comment:
you "can" use `css` via `import {css} from "styled-components";`

It is not needed but better for caching/performance.